### PR TITLE
#1465 Add a pow method in dynawo with a sanity check

### PR DESCRIPTION
**Pull request by rosiereflo**

### CODE REVIEW

**Comment by rosiereflo** [comment]
@gautierbureau @marcochiaramello FYI

**Review by adriguir — COMMENTED** [review]
Thank you for the dev.
1. I think there are a few other existing pow calls in DYNModelLoad in the network that should be updated to pow_dynawo.
2. Shouldn't we move the doubleEq* methods in here? Perhaps later on but would be nice to have a reminder (an open issue)

**Comment by rosiereflo** [comment]
> Thank you for the dev.
> 
>     1. I think there are a few other existing pow calls in DYNModelLoad in the network that should be updated to pow_dynawo.
> 
>     2. Shouldn't we move the doubleEq* methods in here? Perhaps later on but would be nice to have a reminder (an open issue)

1- this is correct and also in the bus if I'm not mistaken, I wasn't sure if I had to update them as this method adds a performance overhead
2- yes why not :)

**Comment by adriguir** [comment]
> > Thank you for the dev.
> > ```
> > 1. I think there are a few other existing pow calls in DYNModelLoad in the network that should be updated to pow_dynawo.
> > 
> > 2. Shouldn't we move the doubleEq* methods in here? Perhaps later on but would be nice to have a reminder (an open issue)
> > ```
> 
> 1- this is correct and also in the bus if I'm not mistaken, I wasn't sure if I had to update them as this method adds a performance overhead
> 2- yes why not :)

I think it would be better for 1. as it can hide some problems otherwise.

File: dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp [hunk 1]
```diff
@@ -23,6 +23,7 @@
 
 #include <boost/algorithm/string/predicate.hpp>
 
+#include "DYNNumericalUtils.h"
```
Comment (adriguir) [inline dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp:26]: I believe the modifications in the core part are still missing, no?

**Review by adriguir — CHANGES_REQUESTED** [review]

File: dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp (hunk 1 above)
Comment (rosiereflo) [reply dynawo/sources/Models/CPP/ModelNetwork/DYNModelBus.cpp:26]: what do you mean by core part?

**Review by adriguir — APPROVED** [review]

**Pull request merged**